<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNew</class>
 <widget class="QDialog" name="AddNew">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>227</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Введите данные аккаунта&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>60</y>
     <width>391</width>
     <height>164</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="website">
        <property name="minimumSize">
         <size>
          <width>121</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Название сайта</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="websiteEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="login">
        <property name="minimumSize">
         <size>
          <width>121</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Логин</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="loginEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="pass">
        <property name="minimumSize">
         <size>
          <width>121</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Пароль</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="passwordEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="mail">
        <property name="minimumSize">
         <size>
          <width>121</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Почта</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="mailEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="phone">
        <property name="minimumSize">
         <size>
          <width>121</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Номер</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="phoneEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="link">
        <property name="minimumSize">
         <size>
          <width>121</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Ссылка</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="linkEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>391</width>
     <height>43</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="minimumSize">
       <size>
        <width>91</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>91</width>
        <height>41</height>
       </size>
      </property>
      <property name="text">
       <string>Подтвердить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="minimumSize">
       <size>
        <width>91</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>91</width>
        <height>41</height>
       </size>
      </property>
      <property name="text">
       <string>Очистить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="minimumSize">
       <size>
        <width>91</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>91</width>
        <height>41</height>
       </size>
      </property>
      <property name="text">
       <string>Отмена</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
